src/ryr-location.html

```diff
@@ -38,59 +37,89 @@
         }
       }
     </style>
-    <google-map id="gmaps" latitude="30.318655" longitude="-97.724454" zoom="15" api-key="[[myApiKey]]" click-events="true">
-      <google-map-marker id="gmarker" latitude="30.318655" longitude="-97.7244549"></google-map-marker>
+
+    <google-map
+      latitude="{{latitude}}"
+      longitude="{{longitude}}"
+      zoom="{{zoom}}"
+      api-key="[[apiKey]]"
+      click-events="true"
+    >
+      <google-map-marker
+        latitude="{{latitude}}"
+        longitude="{{longitude}}"
+      ></google-map-marker>
     </google-map>
-    <div class="container ">
-      <paper-input label="Latitude,longitude " value="{{latlong}} "></paper-input>
-      <paper-icon-button icon="search " title="search "></paper-icon-button>
+
+    <div class="container">
+      <paper-input
+        label="Latitude, longitude"
+        value="{{latlong}}"
+        on-keyup="_onInputChange"
+      ></paper-input>
     </div>
   </template>
-  <!-- Web component class. -->
+
   <script>
     class RyrLocation extends Polymer.Element {
       static get is() {
         return 'ryr-location';
       }
 
       static get properties() {
+        const latCoord = "30.31865599999999";
+        const longCoord = "-97.72445499999999";
```

Comment:
Not a biggie, but these are the coordinates for Epoch Coffee shop in Northloop which I used for my initial test. We should probably use a better landmark like the Texas State Capitol (30.274722,-97.740556), which is also more central.